tasks/configure.yml

```diff
@@ -9,7 +9,7 @@
 - name: Telegraf | Create telegraf.conf (in Python 2)
   template:
     src: telegraf.conf
```

Comment:
my bad. also need to update `src: /etc/telegraf/telegraf-with-python2.conf` in here